x-pack/plugins/apm/public/components/app/error_group_overview/index.tsx

```diff
@@ -110,8 +200,10 @@ export function ErrorGroupOverview() {
           <EuiSpacer size="s" />
 
           <ErrorGroupList
-            items={errorGroupListData.errorGroups}
+            items={items}
             serviceName={serviceName}
+            periods={errorGroupDetailedStatistics}
```

Comment:
It's a little confusing that terms like `item` and `periods` are introduced. Isn't it just new names for "mainStatistics" and "detailedStatistics"?

```suggestion
            serviceName={serviceName}
            mainStatistics={errorGroupMainStatistics}
            detailedStatistics={errorGroupDetailedStatistics}
```